app/models/log-message.js

```diff
@@ -0,0 +1,9 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
```

Comment:
what do you think about using a logging service like https://www.loggly.com/ or something similar? i'm just getting worried about that firebase bill